gulpfile.js

```diff
@@ -15,7 +15,6 @@ gulp.task('test', ['install-azure-account'], (cb) => {
     const env = process.env;
     env.DEBUGTELEMETRY = 1;
     env.CODE_TESTS_WORKSPACE = './test/test.code-workspace';
-    //env.MOCHA_reporter = 'mocha-junit-reporter';
     env.MOCHA_FILE = path.join(__dirname, 'test-results.xml');
```

Comment:
`MOCHA_FILE` is directly related to `mocha-junit-reporter` so might make sense to move that over to index.ts as well

Reply:
done